packages/itwin/measure-tools/src/api/Measurement.ts

```diff
@@ -495,6 +581,20 @@ export abstract class Measurement {
     return true;
   }
 
+  public drawingMetaDataToJSON(): DrawingMetaDataProps | undefined {
```

Comment:
I wouldn't have this as public API on the measurement. Multiple to/from JSON methods are confusing / do not help with discoverability.

For interface types, I usually organize it with a corresponding namespace that has those as functions. E.g.

```
export namespace DrawingMetaData {
 export function toJSON(obj: DrawingMetaData): DrawingMetaDataProps { }

 export function fromJSON(props?: DrawingMetaDataProps): DrawingMetaData | undefined { }
}
```

Also useful for grouping related helper methods and type guards. Although as you can imagine, the more things you cram into a namespace associated with the interface, the more the case for turning it into a class type is made.

Reply:
Should be good now